javascript/javascript1/week1/goodboy-oldboy.js

```diff
@@ -0,0 +1,20 @@
+//Goodboy-Oldboy (A dog age calculator)
+
+const dogYearOfBirth = 2022;
+let dogYearFuture = yearFuture;
+const difference1 = dogYearFuture - dogYearOfBirth;
+const dogYear = 7;
+const humanYear = 1;
+let shouldShowResultInDogYears = true;
+if (shouldShowResultInDogYears === true) {
```

Comment:
Two following options are completely equal, but latter one is shorter and is normally used. AFAIK this should be covered in week2. 

```if (shouldShowResultInDogYears === true) {```

```if (shouldShowResultsInDogYears) {```